src/main/java/com/criticalsoftware/Contact.java

```diff
@@ -1,51 +1,53 @@
 package com.criticalsoftware;
 
-
 import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 
 public class Contact extends PanacheMongoEntity {
 
+    @Email(message = "Email should be valid")
+    @NotBlank(message = "Email may not be blank")
```

Comment:
Cannot be null, also.
@NotNull
Use the same on address, please.

Reply:
The @NotBlank ensures that a String is not null, is not empty, and has at least one visible character.